Misc/NEWS.d/next/Library/2020-12-22-13-16-43.bpo-25246.GhhCTl.rst

```diff
@@ -0,0 +1 @@
+Optimized collections.deque.remove().
```

Comment:
```suggestion
Optimized :meth:`collections.deque.remove`.
```

Reply:
I generally don't put markup in the blurb messages.  They are harder to read as text and harder to test (for valid markup).